src/unix/event.rs

```diff
@@ -0,0 +1,254 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//! This is an abstract EventNotifer and EventConsumer for eventfd and pipefd
+
+use std::io::{Read, Write};
+use std::{
+    fs::File,
+    io,
+    os::fd::{AsRawFd, FromRawFd, IntoRawFd},
+    result,
+};
+
+#[cfg(any(target_os = "linux", target_os = "android"))]
+pub use libc::{EFD_CLOEXEC, EFD_NONBLOCK};
+#[cfg(not(any(target_os = "linux", target_os = "android")))]
+pub use libc::{O_CLOEXEC, O_NONBLOCK};
```

Comment:
Why they are `pub` ?

Reply:
it's not required, just expose the bitflags